src/smt/model.h

```diff
@@ -96,6 +96,14 @@ class Model {
    * is a predicate over t that indicates a property that t satisfies.
    */
   virtual std::vector<std::pair<Expr, Expr> > getApproximations() const = 0;
+  /** get the domain elements for uninterpreted sort s
+   *
+   * This method is gets the interpretation of an uninterpreted sort s.
```

Comment:
```suggestion
   * This method gets the interpretation of an uninterpreted sort s.
```

Reply:
Also, what is `uninterpreted sort s` here -- the given `Type t`? Maybe make this more clear?

Reply:
Yes, meant to say `t`.